rules/windows/collection_posh_screen_grabber.toml

```
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2021/10/19"
+maturity = "production"
+updated_date = "2021/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs.
```

```suggestion
Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs (Remote Access Tools).
```

will need to re-lint for length